### DISCUSSION TITLE
Welcome to Blood-Donation-Project Discussions!

### DISCUSSION THREAD

**Discussion by Saurav-Pant** [Saurav-Pant/Blood-Donation-Project#136] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SaranshBangar** [comment]
Hi everyone, I am Saransh Bangar and I mainly dabble in frontend development (ReactJS, Next.js, JavaScript, TypeScript and TailWind CSS), I am trying to learn a few backend technologies as well to become a complete full stack developer 😁

I have been pretty active in the open source community since the past 4 - 5 months and would love to continue participating and contributing over here in the future!

I wish all the very best to everyone who read this entire thing and wish them a whole lot of success!
